docs/php/how-to-define-workflow-return-values-in-php.md

```diff
@@ -0,0 +1,18 @@
+---
+id: how-to-define-workflow-return-values-in-php
+title: Define Workflow return values in PHP
+sidebar_label: Workflow return values
+---
+
+A Workflow method returns a Generator.
+To properly typecast the Workflow's return value in the client code, use the `#[ReturnType()]` annotation.
+
+```php
+#[WorkflowInterface]
+interface FileProcessingWorkflow {
+
+    #[WorkflowMethod]
+    #[ReturnType("string")]
```

Comment:
Yes, the most basic return type is a native type.

But we also want to show the use of a custom return type being used - typically an object that can be serialized (into json for example) that could be marshalled in other languages.